unixfs/examples/cat.rs

```diff
@@ -0,0 +1,161 @@
+use cid::Cid;
+use ipfs_unixfs::file::{visit::IdleFileVisit, FileReadFailed};
+use std::convert::TryFrom;
+use std::fmt;
+use std::io::{Error as IoError, Read, Write};
+use std::path::PathBuf;
+
+fn main() {
+    let cid = match std::env::args().skip(1).next().map(|s| Cid::try_from(s)) {
```

Comment:
nit: I think `std::env::args().nth(1).map(Cid::try_from)` would work here

Reply:
It does!